include/vinecopulib/misc/implementation/tools_stats.ipp

```diff
@@ -61,6 +62,38 @@ simulate_uniform(const size_t& n,
   return u.unaryExpr([&](double) { return distribution(generator); });
 }
 
+//! @brief Simulates from independendent normals.
+//!
+//! @param n Number of observations.
+//! @param d Dimension.
+//! @param seeds Seeds of the random number generator; if empty (default),
+//!   the random number generator is seeded randomly.
+//!//!
+//! @return An \f$ n \times d \f$ matrix of independent
+//! \f$ \mathrm{N}(0, 1) \f$ random variables.
+inline Eigen::MatrixXd
+simulate_normal(const size_t& n, const size_t& d, std::vector<int> seeds)
```

Comment:
Shouldn't this implementation rely on `simulate_uniform` and `qnorm` ? Then it's literally 2 lines of code, avoid plenty of copy-pasting, offers the quasi-rng possiblity, etc

Reply:
Is this the computational bottleneck?

Reply:
Nah, overhead is small anyway.